.github/workflows/docker.yml

```diff
@@ -95,3 +95,14 @@ jobs:
       if: always() && github.repository == 'openthread/openthread' && github.event_name != 'pull_request'
       run: |
         docker buildx imagetools inspect ${{ steps.prepare.outputs.docker_image }}:${{ steps.prepare.outputs.version }}
+
+    - name: Push README
+      uses: christian-korneck/update-container-description-action@v1
+      if: success() && github.repository == 'openthread/openthread' && github.event_name != 'pull_request'
+      env:
+        DOCKER_USER: ${{ secrets.DOCKER_USERNAME }}
+        DOCKER_PASS: ${{ secrets.DOCKER_PASSWORD }}
+      with:
+        destination_container_repo: openthread/${{ matrix.docker_name }}
+        provider: dockerhub
+        readme_file: 'README.md'
```

Comment:
This is great - it'll be nice to have this auto-updating. There's one add'l field that we might want to add?
```
short_description:
    description: 'Dockerhub only: Set or update the repo short description'
```
maybe:
    description: 'A build environment for use with OpenThread.'

Reply:
I was wondering about that. I believe we have a short description already set on Dockerhub. I'm not sure if leaving this parameter out will leave the existing value unchanged.

Reply:
I'm not sure either. It's not required, but it looks like it defaults to null: 
https://github.com/christian-korneck/update-container-description-action/blob/master/action.yml#L11

Do we have a test environment to try it out first?

Reply:
Is this going to push the main README to all containers on Docker Hub?  Any description probably would need to be generic to all of them.  We could add one to test out what happens.

Reply:
This job is setup as a matrix on https://github.com/openthread/openthread/blob/cf4b71d8b90122f753486215557af46b25bc2185/.github/workflows/docker.yml#L48-L50

So it will only push the README to these repos.

We can certainly merge this first and adjust later as necessary.